### PR TITLE
update/spawners.md

### PR DESCRIPTION
**Pull request by Christiandike**
Hello @sgibson91 @GeorgianaElena could you please help me review this PR?
I'm thankful for your time, as always!

### CODE REVIEW

File: docs/source/reference/spawners.md [hunk 1]
```diff
@@ -69,13 +69,13 @@ via relaxing the `Spawner.start_timeout` config value.
 
 #### Note on IPs and ports
 
-`Spawner.ip` and `Spawner.port` attributes set the _bind_ url,
+`Spawner.ip` and `Spawner.port` attributes set the _bind_ URL,
 which the single-user server should listen on
 (passed to the single-user process via the `JUPYTERHUB_SERVICE_URL` environment variable).
-The _return_ value is the ip and port (or full url) the Hub should _connect to_.
+The _return_ value is the `ip` and `port` (or full URL) the Hub should _connect to_.
```
Comment (sgibson91) [inline docs/source/reference/spawners.md:75]:
```suggestion
The _return_ value is the IP and port (or full URL) the Hub should _connect to_.
```

File: docs/source/reference/spawners.md [hunk 2]
```diff
@@ -297,36 +297,36 @@ Additional variables can be specified via the `Spawner.environment` configuratio
 
 The process environment is returned by `Spawner.get_env`, which specifies the following environment variables:
 
-- JUPYTERHUB*SERVICE_URL - the \_bind* url where the server should launch its http server (`http://127.0.0.1:12345`).
-  This includes Spawner.ip and Spawner.port; _new in 2.0, prior to 2.0 ip,port were on the command-line and only if specified_
+- JUPYTERHUB*SERVICE_URL - the \_bind* URL where the server should launch its HTTP server (`http://127.0.0.1:12345`).
+  This includes `Spawner.ip` and `Spawner.port`; _new in 2.0, prior to 2.0 `ip`, `port` were on the command-line and only if specified_
```
Comment (sgibson91) [inline docs/source/reference/spawners.md:301]:
```suggestion
  This includes `Spawner.ip` and `Spawner.port`; _new in 2.0, prior to 2.0 IP, port were on the command-line and only if specified_
```

**Comment by Christiandike** [comment]
Thank you @sgibson91 - I've applied your suggestions

**Review by sgibson91 — APPROVED** [review]
Thank you @Christiandike - Don't forget to move your handle to the "Done" column in the relevant spreadsheet!

**Pull request merged**